exporter/opentelemetry-exporter-jaeger-proto-grpc/setup.cfg

```diff
@@ -44,7 +44,7 @@ install_requires =
     grpcio >= 1.0.0, < 2.0.0
     googleapis-common-protos ~= 1.52
     opentelemetry-api ~= 1.3
-    opentelemetry-sdk ~= 1.3
+    opentelemetry-sdk ~= 1.12
```

Comment:
@lzchen I was asking about this in the Slack channel. How do specify that this exporter depends on dev changes (which will be part of 1.12)?. I can't do ~= 1.12 because there is no version 1.12 yet and tests fail.

Reply:
The actual correct thing to do is to go back to dev versions to capture dev changes. I believe this was a tactical decision to stop using dev versions since we did not see that many dependency changes after 1.0.0 but that only applied to tracing. Since metrics/logging are part of our actual releases now, that might not be the case. Thoughts @codeboten @ocelotl

Reply:
One way or another we have to do a release that includes the new `InstrumentationScope` class, right? So we could split this PR so that only includes the SDK changes, do another regular release, and then merge the rest of the PR. Can these changes wait until next regular release? If not we in theory could do another regular release right after merging the SDK part of this PR.

Reply:
That sounds like a lot more work than whatever maintenance overhead we tried eliminate by moving away from dev versions.

Reply:
What do you think, @lzchen ?

Reply:
Yeah I think going back to dev versions will be fine. At least until metrics/logs become a bit more stable.